### ISSUE TITLE
Deploy to HoloLens 1 through Build Window Fails after succesful first deploy

### ISSUE THREAD

**Issue by XPav** [microsoft/MixedRealityToolkit-Unity#5697]
Unity 2018.4.6f1
mrtk_development 33fd136c8fcbe7af09cf5fe5c1b92fcaef6430ff

HL1 connected via USB.

To reproduce:
1) Using Build Window, "Build all, then Install."  Everything's good.
2) Make changes
3) Using Build Window, "Build all, then Install."  Builds, does not deploy.

Error in console:
```
NullReferenceException: Object reference not set to an instance of an object
Microsoft.MixedReality.Toolkit.WindowsDevicePortal.DevicePortal+<EnsureAuthenticationAsync>d__39.MoveNext () (at Assets/MixedRealityToolkit/Utilities/WindowsDevicePortal/DevicePortal.cs:822)
--- End of stack trace from previous location where exception was thrown ---
System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
Microsoft.MixedReality.Toolkit.WindowsDevicePortal.DevicePortal+<GetAllInstalledAppsAsync>d__26.MoveNext () (at Assets/MixedRealityToolkit/Utilities/WindowsDevicePortal/DevicePortal.cs:328)
--- End of stack trace from previous location where exception was thrown ---
System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
Microsoft.MixedReality.Toolkit.WindowsDevicePortal.DevicePortal+<GetApplicationInfoAsync>d__25.MoveNext () (at Assets/MixedRealityToolkit/Utilities/WindowsDevicePortal/DevicePortal.cs:308)
--- End of stack trace from previous location where exception was thrown ---
System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
Microsoft.MixedReality.Toolkit.WindowsDevicePortal.DevicePortal+<IsAppInstalledAsync>d__23.MoveNext () (at Assets/MixedRealityToolkit/Utilities/WindowsDevicePortal/DevicePortal.cs:249)
--- End of stack trace from previous location where exception was thrown ---
System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter`1[TResult].GetResult () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
Microsoft.MixedReality.Toolkit.Build.Editor.BuildDeployWindow+<UninstallAppOnTargetDeviceAsync>d__86.MoveNext () (at Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployWindow.cs:1447)
--- End of stack trace from previous location where exception was thrown ---
System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.GetResult () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
Microsoft.MixedReality.Toolkit.Build.Editor.BuildDeployWindow+<InstallOnTargetDeviceAsync>d__82.MoveNext () (at Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployWindow.cs:1363)
--- End of stack trace from previous location where exception was thrown ---
System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.TaskAwaiter.GetResult () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
Microsoft.MixedReality.Toolkit.Build.Editor.BuildDeployWindow+<BuildAll>d__71.MoveNext () (at Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployWindow.cs:1105)
--- End of stack trace from previous location where exception was thrown ---
System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__6_0 (System.Object state) (at <df7127ba07dc446d9f5831a0ec7b1d63>:0)
UnityEngine.UnitySynchronizationContext+WorkRequest.Invoke () (at C:/buildslave/unity/build/Runtime/Export/UnitySynchronizationContext.cs:115)
System.Reflection.MethodBase:Invoke(Object, Object[])
Microsoft.MixedReality.Toolkit.Utilities.SyncContextUtility:ExecuteContinuations() (at Assets/MixedRealityToolkit/Utilities/Async/Internal/SyncContextUtility.cs:53)
UnityEditor.EditorApplication:Internal_CallUpdateFunctions()

```

Restarting Unity fixes the problem.

**Comment by wiwei** [comment]
@XPav what version of VS are you using?

**Comment by XPav** [comment]
VS 2017 15.9.15.

**Comment by keveleigh** [comment]
Should be fixed via #6698
